charts/routes/values.yaml

```diff
@@ -156,6 +156,10 @@ mlp:
 https:
   # NOTE: Set https to true by default to allow google oauth login, disable from parent chart
   enabled: true
+  certificateIssuer:
+    create: true
+    name: default
+    kind: ClusterIssuer
```

Comment:
Should we put `name` and `kind` under a subfield like `external`? Else it might mean we are creating a certificateIssuer with `name` default and `kind` ClusterIssuer

Reply:
makes sense